src/smartcontract/multisig_wallet.scilla.js

```diff
@@ -0,0 +1,407 @@
+const CODE = `scilla_version 0
```

Comment:
@ashishnair-zil is this change supposed to be here or this is from another PR/commit?